policyengine_us/parameters/gov/states/ny/tax/income/credits/geothermal_energy_system/cap.yaml

```diff
@@ -0,0 +1,14 @@
+description: New York caps the geothermal energy system credit at this amount.
+metadata:
+  unit: currency-USD
+  period: year
+  label: New York geothermal energy system equipment credit cap
+  reference:
+    - title: New York State Law § 606 (g)(2)(C)(9)(g-4)
+      href: https://www.nysenate.gov/legislation/laws/TAX/606
+    - title: New York State Department of Taxation and Finance, Geothermal energy systems credit
+      href: https://www.tax.ny.gov/pit/credits/geothermal-energy-system-credit.htm
+
+values:
+  0000-01-01: 0
```

Comment:
```suggestion
```

Reply:
we dont need the 000 values in the parameter files -> we will add this credit to the net income tree starting in 2022

Reply:
I thought you mentioned something of the like but removing 0000 then pre-2022 period would default to values specified for post-2022 (i.e. failing the last test where credit should not be kicking in yet pre-program start date)?